src/utils/ModelUtils.js

```diff
@@ -272,3 +285,13 @@ export function maskPassword(pass) {
 
   return '*'.repeat(pass.length);
 }
+
+// check theRole for example COMPUTE-ROLE matches rolesLimit for example
+// COMPUTE
+export function matchRolesLimit(theRole, rolesLimit) {
+  let match = rolesLimit.some(
+    role =>
+      theRole.includes(role.toUpperCase()) || theRole.includes(role.toLowerCase())
```

Comment:
What is going on here with the case of these roles? IIRC, these are not being entered by the user, so they should never differ in case. If turns out that roles are some special thing that can have a variety of cases and still be considered equal, then we should have some utility method for comparing them and use that everywhere in our code base.

Reply:
Hmmm, my understanding is role could be defined by user...so case could be various